src/components/CartSummary/CartSummary.css

```diff
@@ -0,0 +1,48 @@
+.price {
+  border: 1px solid var(--secondary-color);
+  text-transform: capitalize;
+  box-shadow: 5px 5px var(--gray);
+  width: 15rem;
+  padding: 1rem;
+  margin-left: 1rem;
+  height: 20rem;
+}
+.price-heading,
+.price-inner {
+  font-size: 1.1rem;
+  font-weight: light;
+}
+
+.price-heading,
+.price-total {
+  border-bottom: 1px solid var(--secondary-color);
+  color: var(--secondary-color);
+  font-weight: light;
+}
+.price-desc {
+  display: flex;
+  justify-content: space-between;
+}
+.price-desc-desc,
+.price-desc-price {
+  display: inline;
+  margin: 5px 0;
```

Comment:
In your code, sometimes you use rem for margin and sometimes px. Why?

Reply:
for less than 1 rem we should use px, many reviewrs suggested me this